### PR TITLE
List more optional requirements

### PR DESCRIPTION
**Pull request by s-t-e-v-e-n-k**
Currently, the README does not list enough requirements to have a
passing test suite. To prevent confusion, document them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,12 @@ Mandatory:
 - binutils, cpio, gzip, bzip, xz and zstd
 
 Optional:
+- checkbashisms
```
Comment (Conan-Kudo) [inline README.md:34]: This is part of `devscripts` in most distributions.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (s-t-e-v-e-n-k) [reply README.md:34]: Replace this with devscripts, or "devscripts, or some package that provides checkbashisms" ?

Comment (marxin) [reply README.md:34]: Please use `devscripts`.

**Review by marxin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -133,4 +137,4 @@ The location of `rpmlintrc` can be set using `--rpmlintrc` option. Or you can ha
 `*-rpmlintrc` file in the current working directory.  The best practice is to store the name in `$PACKAGE_NAME.rpmlintrc`.
 
 `setBadness` overrides a default badness for a given check and `addFilter` ignores all errors
-that match the given regular expression.
\ No newline at end of file
+that match the given regular expression.
```
[the remark this answers is not in the document]
Comment (s-t-e-v-e-n-k) [reply README.md:140]: My editor did it, complaining about no newline at the end of the file.

**Comment by marxin** [comment]
Thanks for the pull request.

**Pull request merged**